src/probe_modules/module_bacnet.c

```diff
@@ -73,7 +73,8 @@ int bacnet_init_perthread(void *buf, macaddr_t *src,
 }
 
 int bacnet_make_packet(void *buf, ipaddr_n_t src_ip, ipaddr_n_t dst_ip,
-				uint32_t *validation, int probe_num, UNUSED void *arg)
+				uint32_t *validation, int probe_num, UNUSED void *arg,
+				UNUSED int *len)
```

Comment:
With the caveat that I'm quite unfamiliar with this codebase: I agree with @zakird here. Though that length may be infrequently used, keeping the buffer pointer and its length as close together as possible should reduce potential for errors.

Re; the name: `buf_len` seems perfectly self-explanatory to me, especially if it's located adjacent to `buf`.